contracts/UFragments.sol

```diff
@@ -76,6 +76,34 @@ contract UFragments is ERC20Detailed, Ownable {
     // it's fully paid.
     mapping(address => mapping(address => uint256)) private _allowedFragments;
 
+    // EIP-2612: permit – 712-signed approvals
+    // https://eips.ethereum.org/EIPS/eip-2612
+    bytes public constant EIP712_REVISION = "1";
```

Comment:
` = bytes('1');` to be explicit? This is what AToken does.

Reply:
that doesn't work in solidity 0.4.24 :/